src/main/java/ch/njol/skript/expressions/ExprAnyOf.java

```diff
@@ -0,0 +1,57 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.expressions.base.WrapperExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.LiteralUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Any Of")
+@Description({
+	"Returns an 'or list' composed of the given objects. For example, `any of (1, 2, and 3)` is equivalent to `1, 2, or 3`",
+	"Useful when doing comparisons with variable lists."
+})
+@Example("if any of {_numbers::*} are 1:")
+@Example("if any of {teamA::*} are within location(0, 0, 0) and location(10, 10, 10):")
+public class ExprAnyOf extends WrapperExpression<Object> {
+
+	static {
+		Skript.registerExpression(ExprAnyOf.class, Object.class, ExpressionType.PATTERN_MATCHES_EVERYTHING, "any of [the] %objects%");
```

Comment:
Do you think `one of [the] %objects%` makes sense? Taking from your tests: `assert 1 is one of the integers from 1 to 3`. Might not fit great with some other situations though.

Reply:
I think that fits well too